### PR TITLE
Update ReadMe

### PR DESCRIPTION
**Pull request by DDeAlmeida**
Hi, after using ReadMe to manage the Guildnet explorer Indexer, i saw some modification required.
@frol if you agree with this modifications ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
...
-    $ sudo apt install libpq-dev
+     sudo apt install libpq-dev
     ```
+ * Cargo dependency
+ * 
+   On Debian/Ubuntu:
+    
+    ```bash
+     sudo apt install cargo
+    ```
```
Comment (frol) [inline README.md:39]: Installing Rust compiler is the first item on the list. Rustup is more flexible way to install and manage rustc/cargo versions

File: README.md [hunk 2]
```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
     On Debian/Ubuntu:
     
     ```bash
-    $ sudo apt install libpq-dev
+     sudo apt install libpq-dev
```
Comment (frol) [inline README.md:31]: Well, it is a commit practice to use `$` to indicate that the command should be run on behalf of a regular user.

File: README.md [hunk 3]
```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
...
+```   
 
-Setup PostgreSQL database, create a database with the regular tools, and note the connection string (database host, credentials, and the database name).
+Create a database with the regular tools (Change user and db_name with yours)
+    ```bash
+     sudo su postgres
+     psql
+     create database db_name with owner=user;
+    ```
+Note the connection string (database host, credentials, and the database name).
```
Comment (frol) [inline README.md:55]: Meh, setting up a database is a complicated topic, so I would avoid giving any instructions. Consider finding a good article out there and link it from this README

**Review by frol — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DDeAlmeida) [reply README.md:39]: Yep but Cargo is mandatory for install diesel_cli
So he must be added to the list

File: README.md (hunk 2 above)
Comment (DDeAlmeida) [reply README.md:31]: Oh ok, but this is a pain when copy/paste.
Sorry for this commit.

File: README.md (hunk 3 above)
Comment (DDeAlmeida) [reply README.md:55]: Or we can just give it as a example. as you want

File: README.md (hunk 1 above)
Comment (frol) [reply README.md:39]: cargo gets installed when you install Rustup. It is really hard to install Rust without cargo. Thus, there is no need for this extra step.

File: README.md [hunk 4]
```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
...
 ```bash
-$ cargo install diesel_cli --no-default-features --features "postgres"
+ cargo install diesel_cli --no-default-features --features "postgres"
 ```
 
 And apply migrations
 
 ```bash
-$ diesel migration run
+ diesel migration run
```
Comment (OS-M) [inline README.md:80]: I think there should be "cargo diesel migration run". 
On my system (Ubuntu 21) "diesel migration run" ends up with "command not found" but my variant works fine

Comment (khorolets) [reply README.md:80]: Even after this command?

```
$ cargo install diesel_cli --no-default-features --features "postgres"
```

Comment (OS-M) [reply README.md:80]: Whoops my command doesnt work as well. 
`cargo install diesel_cli --no-default-features --features "postgres"` 
says 
`Ignored package `diesel_cli v1.4.1` is already installed, use --force to override`
but
`diesel migration run`
ends up with
`diesel: command not found`

Comment (OS-M) [reply README.md:80]: The problem was that i didnt have .cargo/bin in my PATH

**Comment by OS-M** [comment]
I would also suggest removing 

> NB! According to changes in nearcore config generation we don't fill all the necessary fields in the config file. While this issue is open https://github.com/near/nearcore/issues/3156 you need to download config you want and replace the generated one manually.

Because mentioned issue is already closed

File: README.md [hunk 5]
```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
...
+ cargo build --release
 ```
 
 ### Configure NEAR Indexer for Explorer
 
 To connect NEAR Indexer for Explorer to the specific chain you need to have necessary configs, you can generate it as follows:
 
 ```bash
-$ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
+ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
```
Comment (OS-M) [inline README.md:94]: As I see in current version the `./target/release/near-indexer` is renamed into `./target/release/indexer-explorer`

**Pull request closed without merging**